.github/workflows/build_wheel.yml

```diff
@@ -0,0 +1,95 @@
+name: Build wheels
+
+on:
+  push:
+    branches:
+      - main  # just build the sdist & wheel, skip release
+    tags:
+      - "*"
+  pull_request:  # also build on PRs touching this file
+    paths:
+      - ".github/workflows/build_wheel.yml"
+      - "ci/*"
+      - "MANIFEST.in"
+      - "pyproject.toml"
+      - "setup.py"
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-2019]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Cache vcpkg
+        uses: actions/cache@v1
+        id: vcpkgcache
+        with:
+          path: C:/vcpkg/installed
+          key: ${{ runner.os }}-vcpkg
+
+      - name: Install GDAL
+        run: |
+          vcpkg install gdal[core]:x64-windows
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.3.1
+        # env:
+        #   CIBW_SOME_OPTION: value
+
+      - uses: actions/upload-artifact@v2
+        with:
+          path: ./wheelhouse/*.whl
+
+  test_wheels:
+    name: windows py${{ matrix.python-version }} tests
+    runs-on: windows-latest
+    needs: build_wheels
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.8', '3.9', '3.10']
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: false
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          # cache: 'pip'
```

Comment:
```suggestion
```

Reply:
I am still going to do another attempt first to get this cache working. If that doesn't work, will clean it up :)